lang/en/settings.php

```diff
@@ -121,6 +121,9 @@
         'actions'   => [
             'update_profile'    => 'Update profile',
         ],
+        'billing_info' => 'Billing Information',
```

Comment:
Refactor to 
```php
'billing' => [
   'title' => 'Billing Information',
   'placeholder' => '....',
   'save' => 'Save billing information',
],
```

So that it has the same "system" as the rest of kanka where things that belong together are in their own array :)